system-variables.md

```diff
@@ -1145,6 +1145,13 @@ MPP is a distributed computing framework provided by the TiFlash engine, which a
 
 </CustomContent>
 
+<CustomContent platform="tidb">
+
+> **Note:**
+>
+> Index acceleration requires a [`temp-dir`](/tidb-configuration-file.md#temp-dir-new-in-v630) that is writable and has enough free space. If the `temp-dir` is unusable, TiDB falls back to non-accelerated index building.
```

Comment:
Is this line duplicated with https://github.com/pingcap/docs/pull/13584/files#diff-ec2cbb115cc58fe3afe2862eb5148732d323cbf7ae35dea97bd381a1f8d0e8a0R1165 ?

Reply:
I've updated this a bit. PTAL @ran-huang